cli/tests/pcluster/validators/test_instances_validators.py

```diff
@@ -499,21 +499,72 @@ def test_instances_allocation_strategy_validator(
     assert_failure_messages(actual_failures, expected_message)
 
 
+# Memory-based scheduling is allowed for Compute Resource that use multiple instance type under 'Instances'
+# but a warning is triggered to inform customers of possible wasted resources.
 @pytest.mark.parametrize(
     "compute_resource_name, instance_types_info, memory_scheduling_enabled, expected_message",
     [
-        # Memory-based scheduling is supported for Compute Resource that use either 'InstanceType' or have a single
-        # instance type under 'Instances'
+        # Memory Diff exceeds both Absolute (2G) and Percentage (0.25) threshold, so Warning is triggered.
```

Comment:
Nitpick: the other day I found out about the [`pytest.param()`](https://github.com/aws/aws-parallelcluster/blob/947f0c6f6b54d11b19a011d668f415b149e35293/cli/tests/pcluster/validators/test_cluster_validators.py#L326) we already use in some unit tests, where we can pass an `id` attribute to describe the case instead of relying on comments. The `id` gets rendered in the stdout/stderr of pytest when running the tests.

Reply:
Cool!

I'll definitely change this.